systest/cluster/cluster.go

```diff
@@ -43,6 +50,19 @@ func WithKeys(n int) Opt {
 	}
 }
 
+// Reuse will try to recover cluster from the given namespace, if not found
+// it will create a new one.
+func Reuse(cctx *testcontext.Context, opts ...Opt) (*Cluster, error) {
+	cl := New(cctx, opts...)
+	if err := cl.reuse(cctx); err != nil {
+		if errors.Is(err, errNotInitialized) {
+			return Default(cctx, opts...)
+		}
+		return nil, err
```

Comment:
are we supposed to use `fmt.Errorf` instead of returning plain errors to the caller?

Reply:
there is no rule to wrap them blindly, i am in general wrapping errors on the module boundary. for example cluster.reuse -> accounts.Recover -> wrapped error